apps/docs/content/components/Prose.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Prose
+status: Experimental
+source: https://github.com/primer/brand/blob/main/packages/react/src/Prose/Prose.tsx
+storybook: '/brand/storybook/?path=/story/components-prose--default'
+description: Use the Prose component to render html content with Primer Brand styles.
+---
+
+```js
+import {Prose} from '@primer/react-brand'
+```
+
+## Examples
+
+<Note>
+
+`Prose` is designed to take sanitized HTML as a string and render it with Primer Brand styles. It is not intended to be used with JSX. The `Prose` component doens't sanitize the data internally. It is the responsibility of the consumer to sanitize the data before passing it to the component.
+
+</Note>
+
+### Default
+
+```jsx live
+<Prose
+  rawHtmlMarkup={`
+	<h1>The tools you need to build what you want.</h1>
+    <p>Contribute to projects quickly with automatic environment setup.</p>
+    <p>Make sure you see the changes you care about.</p>
+    <p>Build community around your code.</p>
+
+    <img
+      src="https://via.placeholder.com/600x400/f5f5f5/f5f5f5.png"
+      alt="placeholder, blank area with an off-white background color"
+    />
+
+    <h2>Automation and CI/CD</h2>
+    <ul>
+      <li>
+        Automate everything: CI/CD, testing, planning, project management, issue labeling, approvals, onboarding, and
+        more.
+      </li>
+      <li>Standardize and scale best practices, security, and compliance across your organization.</li>
+      <li>Get started quickly with thousands of actions from partners and the community.</li>
+    </ul>
+
+    <h2>Security</h2>
+    <p>
+      Secure code as you write it. Automatically review every change to your codebase and identify vulnerabilities
+      before they reach production. <a href="/#">Learn more here.</a>
+    </p>
+`}
+/>
```

Comment:
As this is the main example, I think we could expand but genericize the content a little to make it appear use-case agnostic, while highlighting depth of support (showcase more scenarios).

E.g for additional content:


https://github.com/primer/brand/assets/13340707/9369fac7-ef32-4cdf-8f7a-4a825d89d7a6

Few things we should consider adding:

- all heading levels displayed in a representative way, excluding h1 as that's typically outside Prose scope
- video example
- ordered list
- `kbd` support (missing styles, but I think this will come up frequently)
- `blockquote`  (missing styles, but I think this will come up frequently)
- 

Here's the code from the example:

```suggestion
<Prose
  rawHtmlMarkup={`
    <h2>Heading level 2</h2>
    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut tempor lectus ipsum, consectetur convallis diam pretium quis. Proin ut felis ut eros tristique tincidunt.</p>
    <blockquote>Nulla ac odio eu magna hendrerit porta. Donec nec eros quis tortor tincidunt vulputate. Aenean id pharetra diam, sit amet auctor leo. Aliquam erat volutpat.</blockquote>
    <p>Integer pellentesque pretium nulla viverra molestie. Praesent quis pretium sapien. Sed convallis eget lectus et pulvinar:</p>
    <ul>
      <li>
        Vivamus eu risus nec lectus consequat rutrum at vel lacus.
      </li>
      <li>Donec at dolor ut metus imperdiet congue vel porta nunc.
      </li>
      <li>Quisque eu tortor suscipit, congue quam in, bibendum tellus.</li>
    </ul>
    <h3>Heading level 3</h3>
    <p>Pellentesque non ornare ligula. Suspendisse nibh purus, pretium id tortor sit amet, tincidunt gravida augue. Ut malesuada, nisl vel dignissim mollis, <kbd>Ctrl</kbd>+<kbd>F</kbd> shortcut (⌘+<kbd>F</kbd> on Mac) mi dui molestie erat, nec laoreet est lectus ac urna. </p>
    <img
      src="https://via.placeholder.com/600x400/f5f5f5/f5f5f5.png"
      alt="placeholder, blank area with an off-white background color"
      />
    <h4>Heading level  4</h4>
    <p>
      Secure code as you write it. Automatically review every change to your codebase and identify vulnerabilities
      before they reach production. <a href="/#">Learn more here.</a>
    </p>
    <h5>Heading level 5</h5>
    <ol>
      <li>
        Vivamus eu risus nec lectus consequat rutrum at vel lacus.
      </li>
      <li>Donec at dolor ut metus imperdiet congue vel porta nunc.
      </li>
      <li>Quisque eu tortor suscipit, congue quam in, bibendum tellus.</li>
    </ol>
    <h6>Heading level 6</h6>
    <p>Pellentesque non ornare ligula. Suspendisse nibh purus, pretium id tortor sit amet, tincidunt gravida augue.</p>
    <p><iframe width="100%" height="315" src="https://www.youtube.com/embed/4RfD5JiXt3A" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe></p>
    <p>Nunc velit odio, posuere eu felis eget, consectetur fermentum nisi. Aenean tempor odio id ornare ultrices. Quisque blandit condimentum tellus, semper efficitur sapien dapibus nec. </p>
    `}
/>
```